### PR TITLE
feat: load package configuration file and upload it with enterprisePackage target, MISC-446

### PR DESCRIPTION
**Pull request by Obero**
Depends on https://github.com/gatling/gatling-enterprise-plugin-commons/pull/113/files

### CODE REVIEW

**Review by Isammoc — APPROVED** [review]

**Comment by slandelle** [comment]
I'm doing a PR with a new version of my `wip` branch with squashed commits.

File: pom.xml [hunk 1]
```diff
@@ -78,6 +79,11 @@
             <version>${maven3.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${apache-commons-version}</version>
```
Comment (slandelle) [inline pom.xml:85]: Are you adding a new dependency just for StringUtils.isNotBlank which is just `string != null && !string.trim().isEmpty()`?

File: src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java [hunk 2]
```diff
@@ -51,6 +54,21 @@ public final class EnterpriseUploadMojo extends AbstractEnterprisePluginMojo {
 
   @Override
   public void execute() throws MojoFailureException {
+    final BatchEnterprisePlugin enterprisePlugin = initBatchEnterprisePlugin();
+
+    final String projectDirectory = mavenProject.getBasedir().getAbsolutePath();
+    final String jsonConfig = PackageConfiguration.loadToJson(projectDirectory);
+
+    if (StringUtils.isNotBlank(jsonConfig)) {
```
Comment (slandelle) [inline src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java:62]: This clean up should be done in `PackageConfiguration.loadToJson`. Client code should only have to test `null`.

**Review by slandelle — CHANGES_REQUESTED** [review]

File: src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java (hunk 2 above)
Comment (slandelle) [reply src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java:62]: https://github.com/gatling/gatling-enterprise-plugin-commons/pull/114

Comment (slandelle) [reply src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java:62]: Please upgrade to commons 1.9.0-M7 so you can only test against null.

Comment (Obero) [reply src/main/java/io/gatling/mojo/EnterpriseUploadMojo.java]: Done

**Review by slandelle — APPROVED** [review]

**Pull request merged**